test/e2e/util_test.go

```diff
@@ -548,7 +562,7 @@ func createInvalidGRPCCatalogSource(crc versioned.Interface, name, namespace str
 	catalogSource, err := crc.OperatorsV1alpha1().CatalogSources(namespace).Create(context.Background(), catalogSource, metav1.CreateOptions{})
 	Expect(err).ToNot(HaveOccurred())
 	ctx.Ctx().Logf("Catalog source %s created", name)
-	return catalogSource, buildCatalogSourceCleanupFunc(crc, namespace, catalogSource)
+	return catalogSource, buildCatalogSourceCleanupFunc(nil, crc, namespace, catalogSource)
```

Comment:
This is an unfortunate side effect of a good chunk of testing helper functions that should be refactored at some point, but we could also just update this overall function to take a `c operatorclient.ClientInterface` parameter, and at the call sites, you can get access to an already initialized client through `newKubeClient()`.

Reply:
Thanks.  I change to use newKubeClient() in the helper function instead of taking the client as an argument.

Reply:
Sounds good - we'll eventually need to refactor those helper function signatures, but this should (hopefully) cut down on e2e flakes in the interim.